docker/docker.go

```diff
@@ -22,201 +22,467 @@ limitations under the License.
 package docker
 
 import (
-	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
-	"path/filepath"
+	"strconv"
 	"strings"
 	"time"
 
-	"github.com/opencontainers/runc/libcontainer/cgroups"
-
 	"github.com/intelsdi-x/snap-plugin-collector-docker/client"
-	tls "github.com/intelsdi-x/snap-plugin-collector-docker/tools"
 	"github.com/intelsdi-x/snap-plugin-collector-docker/wrapper"
 	"github.com/intelsdi-x/snap/control/plugin"
 	"github.com/intelsdi-x/snap/control/plugin/cpolicy"
 	"github.com/intelsdi-x/snap/core"
+
+	dock "github.com/fsouza/go-dockerclient"
+	utils "github.com/intelsdi-x/snap-plugin-utilities/ns"
 )
 
 const (
 	// namespace vendor prefix
```

Comment:
I've just kept this as it is on master branch (https://github.com/intelsdi-x/snap-plugin-collector-docker/blob/master/docker/docker.go)